src/js/common/helpers/login-helpers.js

```diff
@@ -80,3 +80,21 @@ export function handleLogin(loginUrl, onUpdateLoginUrl) {
   }
   return Promise.reject('Could not log in; loginUrl not provided.');
 }
+
+function isGaLoaded() {
+  return !!(window.ga && ga.create);
+}
+
+// google analytics client Id
+/* global gaClientId ga:true */
+export function gaClientId() {
+  let clientId;
+  if (isGaLoaded()) {
```

Comment:
@jbalboni @robertfairhead I _think_ this is sufficient.  We expect that GA is already loaded 99.9% of the time, but this safeguard should prevent any kind of `Uncaught ReferenceError: ga is not defined` JS error.s

Reply:
Looks good to me.